recipes/opengv/all/test_v1_package/test_package.cpp

```diff
@@ -0,0 +1,30 @@
+#include <opengv/math/roots.hpp>
```

Comment:
I take it you didn't mean to duplicate this file?

Reply:
for cmake lists in this folder

https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/autotools_package/all/test_v1_package/CMakeLists.txt

Reply:
I'm not sure if I understand these comments :)

Reply:
Can you delete `test_v1_package/test_package.cpp` and re-use the file `test_package/test_package.cpp` in `test_v1_package` by using a relative path to compile it?

Reply:
Oh I thought you were talking abut the header. The way github render its makes it looks like you're talking about `roots.hpp`.
Anyways this totally makes sense. I was sure I was reusing the cpp file from the v2 package, maybe I forgot to commit that. I'll check,